packages/cli/src/lib/missing-link.ts

```diff
@@ -0,0 +1,19 @@
+import {renderFatalError} from '@shopify/cli-kit/node/ui';
+import {outputContent, outputToken} from '@shopify/cli-kit/node/output';
+import type {AdminSession} from '@shopify/cli-kit/node/session';
+
+interface Args {
+  adminSession: AdminSession;
+}
+
+export function logMissingLink({adminSession}: Args) {
```

Comment:
Just a random thought: should we group this kind of logs in a single file for better organization? Maybe in `lib/log.ts` or `lib/render-errors` or similar?

Reply:
I like that idea! A lot nicer than bouncing around between a bunch of files.